MAUI/Masked-Entry/Basic-Features.md

```diff
@@ -0,0 +1,201 @@
+platform: maui
+control: SfMaskedEntry
+documentation: ug
+---
+
+# Basic Features in .NET MAUI Masked Entry (SfMaskedEntry)
+
+## Setting ClearButtonVisibility
+
+ClearButtonVisibility, of type ClearButtonVisibility, controls whether a clear button is displayed, which enables the user to clear the text. The default value of this property is `ClearButtonVisibility.Never`.
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfMaskedEntry WidthRequest="200"
+                          MaskType="Simple"
+                          Mask="(000) 000-0000"
+                          ClearButtonVisibility = "Visible" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+maskedEntry = new SfMaskedEntry();
+maskedEntry.WidthRequest = "200";
+maskedEntry.MaskType = MaskedEntryMaskType.Simple;
+maskedEntry.Mask = "(000) 000-0000";  
+maskedEntry.ClearButtonVisibility = Visible;
+
+{% endhighlight %}
+{% endtabs %}
+
+![ClearButtonVisibility](MaskedEntry_Images/MaskedEntry_ClearButtonVisibility.png)
+
+## Setting CursorPosition
+
+Cursor position in the masked entry can either be obtained or updated using `CursorPosition` property in SfMaskedEntry.
+
+## Setting FontSize
+The Masked Entry control provides the user to customize the font size of the text inside the entry part using `FontSize` property of type double.
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfMaskedEntry WidthRequest="200"
+                          MaskType="Simple"
+                          Mask="(000) 000-0000" Value="1234"
+                          FontSize="18"/>
+
+{% endhighlight %}
+{% highlight C# %}
+
+maskedEntry = new SfMaskedEntry();
+maskedEntry.WidthRequest = "200";
+maskedEntry.MaskType = MaskedEntryMaskType.Simple;
+maskedEntry.Mask = "(000) 000-0000";  
+maskedEntry.Value = 1234;
+maskedEntry.FontSize = 18;   
+
+{% endhighlight %}
+{% endtabs %}
+
+![FontSize](MaskedEntry_Images/MaskedEntry_FontSize.png)
+
+## Setting FontAttributes
+
+The Masked Entry control provides the user to customize the font attribute of the text inside the entry part using `FontAttributes` property of type FontAttributes.
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfMaskedEntry WidthRequest="200"
+                          MaskType="Simple"
+                          Mask="(000) 000-0000" Value="1234"
+                          FontAttributes="Bold"/>
+
+{% endhighlight %}
+{% highlight C# %}
+
+maskedEntry = new SfMaskedEntry();
+maskedEntry.WidthRequest = "200";
+maskedEntry.MaskType = MaskedEntryMaskType.Simple;
+maskedEntry.Mask = "(000) 000-0000";  
+maskedEntry.Value = 1234;
+maskedEntry.FontAttributes = FontAttributes.Bold;   
+
+{% endhighlight %}
+{% endtabs %}
+
+![FontAttributes](MaskedEntry_Images/MaskedEntry_FontAttributes.png)
+
+## Setting FontFamily
+
+The Masked Entry control provides the user to customize the font family of the text inside the entry part using `FontFamily` property of type string.
```

Comment:
The Masked Entry control allows the user to customize the text's font family inside the entry part using the `FontFamily` property of the type string.

Reply:
Addressed the changes.